### DISCUSSION TITLE
Harvester automatically disconnected since updated to v.1.2.0

### DISCUSSION THREAD

**Discussion by fzfc** [Chia-Network/chia-blockchain#7328] [Support]
I updated my farmer(RPi 4B w. Ubuntu 20) and two harvester(Win and RPi 3B w. Ubuntu 20) to version 1.2.0. At the beginning, all harvesters can connect to the farmer normally (via CLI), but after a while both were disconnected automatically. And on both harvesters I got the message "Rate limiting ourselves. message type: respond_plots, peer: xx.xx.x.x" in the log file. Any suggestions?

**Comment by xulifan** [comment]
I am experiencing the sam error.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
